lib/persist.js

```diff
@@ -0,0 +1,36 @@
+const redis = require('redis').createClient(process.env.REDIS_URL);
+const { promisify } = require('util');
+const get = promisify(redis.get).bind(redis); // this maps redis.get(key, callback(results)) to a promise-compatible function
+const moment = require('moment');
+const { getCurrentEmojis } = require('./slack');
+
+const synchronizeEmojis = async () => {
+  // what emojis are present _right now_?
+  const currentEmojis = await getCurrentEmojis();
+
+  // store that list
+  const today = moment();
+  const todayKey = today.format('MM-DD-YYYY');
+  // redis.set(todayKey, JSON.stringify(currentEmojis));
+
+  // what emojis did we have yesterday?
+  const yesterday = moment().subtract(1, 'day');
+  const yesterdayKey = yesterday.format('MM-DD-YYYY');
+  let yesterdaysEmojis = JSON.parse(await get(yesterdayKey));
+  if (Array.isArray(yesterdaysEmojis)) {
```

Comment:
Previous data points are stored as simple arrays of emoji keys, `['wow', 'doh', 'heart']`. This hack converts that array back into the slack-compatible Object response shape. We lose the original definition of the emoji, including url and whether it's an alias, but at least we don't have to parse `today` and `yesterday` differently conditionally downstream.

My hope here is that after this merges and we start collecting _everything_ we can turn this off in awhile, since we don't care about anything other than yesterday/today right now.